code/game/atoms.dm

```diff
@@ -81,6 +81,10 @@
 		reagents = null
 	invisibility = 101
 	return ..()
+	
+//Hook for running code when a dir change occurs
+/atom/proc/setDir(newdir)

```

Comment:
@tigercat2000 he already replaced all the instances of `set_dir` with `setDir`...andd, well, given what @Markolie stated, I'm inclined that keeping our proc names the same as TG's is more beneficial to us and future coders.